README.md

```diff
@@ -1,11 +1,11 @@
 ![](images.jfif)
```

Comment:
[OPTIONAL] Kindly ensure you update your REAME file with the current image of your project, this helps to make your project more descriptive.